stingray/modeling/tests/test_parameterestimation.py

```diff
@@ -53,7 +53,8 @@ def setup_class(cls):
             scipy.stats.norm(loc=cls.a_mean, scale=cls.a_var).pdf(amplitude)
 
         cls.priors = {"amplitude": p_amplitude}
-        cls.lpost = PSDPosterior(cls.ps, cls.model)
+        cls.lpost = PSDPosterior(cls.ps.freq, cls.ps.power, cls.model,
```

Comment:
Why are freq, power and m explicitly passed now, instead of the simple power spectrum?